rel_bball_analytics/players/views.py

```diff
@@ -13,14 +13,20 @@ def search():
         season = request.form["season"]
 
         search_result = get_search_result(name=name, season=season)
+        season_label = f"{season}-{int(season) + 1}"
```

Comment:
Create function in `players/styles.py` to handle this logic for all cases where this is called.